src/dhttest/cases/neo/GetChannels.d

```diff
@@ -62,13 +62,10 @@ public class GetChannels : NeoDhtTestCase
 
     override public void cleanup ( )
     {
-        super.cleanup();
-
-        auto legacy_dht = new Legacy.DhtClient;
-        legacy_dht.handshake(10000);
-
         foreach ( channel; this.channels )
-            legacy_dht.removeChannel(channel);
+            this.dht.blocking.removeChannel(channel);
+
+        super.cleanup();
```

Comment:
Why is this moved after the `removeChannel`?

Reply:
Hmm, not sure if there was a reason for that or if it was a mistake. I can try compiling it without that change.

Reply:
Yeah, the reason is that the last thing `super.cleanup` does is shut down the DHT client. Previously, when this test was using a separate client instance to remove the used channels, this had no effect.

Reply:
Makes sense :+1: